fdbserver/KeyValueStoreMemory.actor.cpp

```diff
@@ -319,7 +319,7 @@ class KeyValueStoreMemory final : public IKeyValueStore, NonCopyable {
 		size_t expectedSize() const { return p1.expectedSize() + p2.expectedSize(); }
 	};
 	struct OpHeader {
-		int op;
+		uint32_t op;
```

Comment:
Can we not change the type?

Reply:
we overflow with a signed int since we're using the first bit

Reply:
There is no risk for backwards compatibility since were going from an int to unsigned int and we never had negative ops

Reply:
okay. with the static_assert above I'm okay with it.